src/Kunstmaan/NodeSearchBundle/Services/SearchService.php

```diff
@@ -67,17 +67,35 @@ public function setDefaultPerPage($defaultPerPage)
     /**
      * @return RenderContext
      */
+    public function getRenderContext()
+    {
+        return $this->renderContext;
+    }
+
+    /**
+     * @param RenderContext $renderContext
+     */
+    public function setRenderContext($renderContext)
+    {
+        $this->renderContext = $renderContext;
+    }
+
+    /**
+     * @return RenderContext
+     * @deprecated Use getRenderContext
+     */
     public function getRenderContect()
     {
-        return $this->renderContect;
+        return $this->renderContext;
```

Comment:
Make it deprecated now, and we will remove all deprecations things before 4.0 gets released.